app/assets/javascripts/discourse/components/choose-topic.js.es6

```diff
@@ -4,14 +4,38 @@ import Component from "@ember/component";
 import discourseDebounce from "discourse/lib/debounce";
 import { searchForTerm } from "discourse/lib/search";
 import { observes } from "discourse-common/utils/decorators";
+import discourseComputed from "discourse-common/utils/decorators";
 
 export default Component.extend({
   loading: null,
   noResults: null,
   topics: null,
+  selectedTopic: null,
   selectedTopicId: null,
   currentTopicId: null,
-  topicTitle: null,
+  additionalFilters: "",
+  topicTitle: "",
+  label: null,
+  loadOnInit: false,
+  topicChangedCallback: () => {},
```

Comment:
This is probably leaking, shouldn't init anything but string, null and booleans here.

Should be set at null here, and do this in init